src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs

```diff
@@ -93,13 +101,15 @@ at com.android.dx.command.Main.main(Main.java:106)
 		protected override bool HandleTaskExecutionErrors ()
 		{
 			if (foundError) {
+				assemblyMap.Load (AssemblyIdentityMapFile);
```

Comment:
Ok, so it only loads this file if there is an error. 👍

Reply:
It only *loads* on error, but it *always* creates the `AssemblyIdentityMap` instance, even if no errors are produced.  Is that necessary?